mulesoft/docs-site-ui
# [W-11799442] remove version selector labels from archive site

ref: W-11799442

in archive site, in the version selectors, remove the labels "current version" and "previous versions" since they don't make sense there.

![Screen Shot 2022-09-22 at 3 52 43 PM](https://user-images.githubusercontent.com/10934908/191865128-d2a6bb67-0a4e-4be2-99ca-95262d04b47f.png)

To test, here are the recommended steps:
- open docs-site-ui, switch to this branch, run `gulp bundle` to create a local UI bundle
- open docs-site-playbook, switch to the archive branch, create a local playbook, change ui.bundle.url to the local UI bundle, run `make gen-site playbook=<playbook path>` to build it, look at the built page and make sure that there are no version labels in the version selectors, and no other UI defects
- still on docs-site-playbook, switch to the default branch, create a local playbook, change ui.bundle.url to the local UI bundle and remove most of the repos, run `make gen-site playbook=<playbook path>` to build it, look at the built page and make sure that the version labels **are present** in the version selectors, and no other UI defects
